trunk/includes/pardot-settings-class.php

```diff
@@ -209,39 +199,35 @@ static function is_admin_page() {
 	 *
 	 * @since 1.0.0
 	 */
-	function admin_menu() {
-		$title = __( 'Pardot', 'pardot' );
-		self::$admin_page = add_options_page( $title, $title, 'manage_options', self::$PAGE , array( $this, 'settings_page' ) );
+	function admin_menu()
+	{
+		$title = __('Pardot', 'pardot');
+		self::$admin_page = add_options_page($title, $title, 'manage_options', self::$PAGE, [$this, 'settings_page']);
 	}
 
 	/**
 	 * Display admin notices if applicable.
 	 *
 	 * @since 1.0.0
 	 */
-	function admin_notices() {
-		if ( ! current_user_can( 'install_plugins' ) || ! current_user_can( 'manage_options' ) ) {
+	function admin_notices()
+	{
+		if (!current_user_can('install_plugins') || !current_user_can('manage_options')) {
 			/**
 			 * If the user can't install plugins or manage options, then of course we should bail!
 			 * No message for you!
 			 */
 			return;
 		}
 
-		if (self::is_authenticated() && self::get_setting('auth_type') == 'pardot') {
-            $msg = __( 'Pardot authentication is being discontinued in February 2021.  To update your authentication to Salesforce SSO, go to your %s.', 'pardot' );
-            $msg = sprintf( $msg, self::get_admin_page_link( array( 'link_text' => __( 'Pardot plugin settings', 'pardot' ) ) ) );
-            echo "<div class=\"updated\" style=\"border-left-color: #ffb900\"><p>{$msg}</p></div>";
-        }
```

Comment:
Old banner message telling people to start using Salesforce SSO by February 2021.  No longer needed.